src/ui/toast.js

```diff
@@ -95,67 +95,65 @@ export class Toast {
   /**
    * Builds the content of the iframe. On load, animates the toast.
    */
-  buildToast_() {
+  async buildToast_() {
     const toastDurationSeconds = 7;
-    return this.activityPorts_
-      .openIframe(this.iframe_, this.src_, this.args_)
-      .then((port) => {
-        return port.whenReady();
-      })
-      .then(() => {
-        resetStyles(this.iframe_, ['height']);
-
-        this.animate_(() => {
-          setImportantStyles(this.iframe_, {
-            'transform': 'translateY(100%)',
-            'opactiy': 1,
-            'visibility': 'visible',
-          });
-          return transition(
-            this.iframe_,
-            {
-              'transform': 'translateY(0)',
-              'opacity': 1,
-              'visibility': 'visible',
-            },
-            400,
-            'ease-out'
-          );
-        });
-
-        // Close the Toast after the specified duration.
-        this.doc_.getWin().setTimeout(() => {
-          this.close();
-        }, (toastDurationSeconds + 1) * 1000);
+    const port = await this.activityPorts_.openIframe(
+      this.iframe_,
+      this.src_,
+      this.args_
+    );
+    await port.whenReady();
+    resetStyles(this.iframe_, ['height']);
+
+    this.animating_ = this.animate_(() => {
+      setImportantStyles(this.iframe_, {
+        'transform': 'translateY(100%)',
+        'opactiy': 1,
```

Comment:
Typo for `opacity` (already existing issue).

Reply:
Nice catch, fixed